community/neo4j/src/test/java/migration/Start3_2DbOn3_3AndCreateFusionIndexIT.java

```diff
@@ -174,6 +175,20 @@ private int countIndexedNodes( GraphDatabaseAPI db, Label label, String... keys
                 }
 
                 CapableIndexReference index = ktx.schemaRead().index( labelId, propertyKeyIds );
+
+                // wait for index to come online
+                final long timeout = 5000;
+                final long start = System.currentTimeMillis();
+                long runningFor;
+                InternalIndexState state;
+                do
+                {
+                    state = ktx.schemaRead().indexGetState( index );
+                    runningFor = System.currentTimeMillis() - start;
+                }
+                while ( state != InternalIndexState.ONLINE && runningFor < timeout );
+
+
```

Comment:
There is also `db.schema().awaitIndexesOnline(…)`.

Reply:
I think I don't have access to that from this IT, do I?

Reply:
Yeah, let me use that one, I haven't seen that the GraphDatabaseAPI was in scope

Reply:
I tried, but somehow that does not actually seem to wait for that index...